lib/apiservers/engine/backends/container/viccontainer.go

```diff
@@ -15,9 +15,15 @@
 package container
 
 import (
+	"fmt"
+	"strings"
+
 	containertypes "github.com/docker/engine-api/types/container"
 )
 
+// DefaultEnvPath defines the default PATH environment variable to use in a container
```

Comment:
It comes from Docker's default but we may want to change that in the future, so I didn't bother referencing that. Additionally, it's such a common set of bin directories that I don't think anyone would wonder why these are the defaults.